kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -280,6 +282,7 @@ def exec_foundry_prove(
     implication_every_block: bool = True,
     rpc_base_port: Optional[int] = None,
     bug_report: bool = False,
+    rpc_command: Optional[str] = None,
```

Comment:
We can probably set the default here, and simplify the handling logic? `rpc_command: str = 'kore-rpc',`?

Reply:
I want to see @tothtamas28 opinion on how to do this though.

Reply:
I was trying to keep `Foundry.foundry_prove` behaviour unchanged by default... maybe not necessary, though.
Happy to follow any other suggestions. What we need to keep is the second default in `utils.py:parallel_config_explore`, since that function is being called from `exec_prove`, too.

Reply:
For parameters that inherit the default value of a called function I like to go with `Optional`:

I.e. in that case I prefer this:

```python3
def foo(x=None):
    return bar(x)

def bar(x=None):
    x = 0 if x is None else x
    return x
```

Over this:

```python3
def foo(x=0):
    return bar(x)

def bar(x=0):
    return x
```

(It gets worse as nesting increases.)

(Also, there are cases when exposing the default value is not  an option / is not advised, e.g. if the value is mutable.)

Reply:
OK, so IIUC the code can stay as it is now (setting defaults later rather than earlier, using `Optional` while going through the call stack)